### PR TITLE
test(e2e,playwright): form-helper component

### PR DESCRIPTION
**Pull request by shabeebk**
With this commit, the playwright will test the  form-helper component for possible form views w/ and w/o github link.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705

### CODE REVIEW

File: e2e/form-helper.spec.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
```
Comment (naomi-lgbt) [inline e2e/form-helper.spec.ts:8]:
```suggestion
```

We don't need this - playwright does it automatically.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by shabeebk** [comment]
Thanks for the review @naomi-lgbt. Applied the suggestion.

**Comment by shabeebk** [comment]
I had to fix the lint and re-push the branch.

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @shabeebk 🎉 Thanks for contributing to freeCodeCamp and happy hacktoberfest 🎉

**Comment by shabeebk** [comment]
Thanks team.

**Pull request merged**